tutorials/cloudsdk-e2e-test-secure-applications/cloudsdk-e2e-test-secure-applications.md

```diff
@@ -10,9 +10,9 @@ primary_tag: products>sap-s-4hana-cloud-sdk
 ---
 
 ## Prerequisites
-  - [Set up CI/CD](https://developers.sap.com/tutorials/cloudsdk-ci-cd.html)
-  - [End to end test](https://developers.sap.com/tutorials/cloudsdk-e2e-testing.html)
-  - [Secure your application](https://developers.sap.com/tutorials/s4sdk-secure-cloudfoundry.html)
+  - [Set up CI/CD](cloudsdk-ci-cd)
```

Comment:
@thecodester These changes are already merged to master. Not sure how they made it into this PR.

Reply:
Fixed by cleaning up the git history.